lib/jsontableschema.rb

```diff
@@ -28,6 +28,7 @@
 require "jsontableschema/types/string"
```

Comment:
I suppose it's something like `__init__.py` in Python? I'm not sure how it works in Ruby but in Python we use this to declare API - adding only public interface and not adding internal stuff (like types after field introduction). Like it's internal API could be changed anytime without any notification etc.